spanner/test/spannerTest.php

```diff
@@ -931,11 +941,54 @@ public function testDmlReturningDelete()
      */
     public function testAddDropDatabaseRole()
     {
-        $output = $this->runFunctionSnippet('add_drop_database_role');
+        $output = $this->runFunctionSnippet('add_drop_database_role',
+        [self::$instanceId,
+        self::$databaseId,
+        self::$databaseRole]);
         $this->assertStringContainsString('Waiting for create role and grant operation to complete... ' . PHP_EOL, $output);
         $this->assertStringContainsString('Created roles new_parent and new_child and granted privileges ' . PHP_EOL, $output);
         $this->assertStringContainsString('Waiting for revoke role and drop role operation to complete... ' . PHP_EOL, $output);
-        $this->assertStringContainsString('Revoked privileges and dropped roles new_child and new_parent ' . PHP_EOL, $output);
+        $this->assertStringContainsString('Revoked privileges and dropped role new_child ' . PHP_EOL, $output);
+    }
+
+    /**
+     * @depends testAddDropDatabaseRole
+     */
+    public function testListDatabaseRoles()
+    {
+        $output = $this->runFunctionSnippet('list_database_roles',
+        [self::$projectId,
+        self::$instanceId,
+        self::$databaseId]);
+        $this->assertStringContainsString(sprintf('databaseRoles/%s', self::$databaseRole), $output);
+    }
+
+    /**
+     * @depends testAddDropDatabaseRole
+     * @depends testInsertDataWithDml
+     */
+    public function testReadDataWithDatabaseRole()
+    {
+        $output = $this->runFunctionSnippet('read_data_with_database_role',
+        [self::$instanceId,
+        self::$databaseId,
+        self::$databaseRole]);
+        $this->assertStringContainsString('SingerId: 10, Firstname: Virginia, LastName: Watson', $output);
+    }
+
+    /**
+     * depends testAddDropDatabaseRole
+     */
+    public function testEnableFineGrainedAccess()
+    {
+        self::$serviceAccountEmail = $this->createServiceAccount(str_shuffle('testSvcAcnt'));
+        $output = $this->runFunctionSnippet('enable_fine_grained_access',
+        [self::$projectId,
+        self::$instanceId,
+        self::$databaseId,
+        self::$databaseRole,
+        sprintf('serviceAccount:%s', self::$serviceAccountEmail)]);
+        $this->assertStringContainsString('Enabled fine-grained access in IAM.', $output);
```

Comment:
If you accept suggestion above, this needs to be changed:

```suggestion
        $this->assertStringContainsString('Enabled fine-grained access in IAM', $output);
```

Reply:
Done